<launch>
  <!-- Launch openpose service-->
  <node name="openpose_ros_node_3d" pkg="openpose_ros_pkg" type="openpose_ros_node_3d" output="screen" />
  <!-- Launch the skeleton extractor node -->
  <node name="skeleton_extract_3d_node" pkg="skeleton_extract_3d" type="skeleton_extract_3d_node" />
    <remap from="point_cloud" to="/zed2/zed_node/point_cloud/cloud_registered" />
    <remap from="image" to="/zed2/zed_node/rgb/image_rect_color" />
    <remap from="skeleton_2d_detector" to="detect_poses_3d" />

</launch>
